### ISSUE TITLE
Chemical implants can be activated by action button

### ISSUE THREAD

**Issue by Uqualm** [HippieStationCode/HippieStation13#1857]
The original purpose of chemical implants when they were first created was to allow Security to load up a problem criminal or suspicious person with a chemical implant to dissuade them from further action. If they acted up, the Warden would activate the implant (which was usually filled with a lethal injection) via a computer console in the brig to punish them. At some point, /tg/ reworked the way implants work, adding action buttons for remote activation of implants rather than emoting. Unfortunately this functionality extended to all implants, even chemical ones. This has created two balance issues.

1.Criminals can activate their lethal implant at any time. This typically happens in Medbay surrounded by doctors who immediately heal the implanted person, negating the implant. 

2.This is the big one. Security and power gamers will load up on implants filled with chemicals that prevent them from being killed except through the most exceptional and unavailable means. It is activated when the implanted person is losing a fight and suddenly they're invincible. This is no different then someone using an href exploit to give themselves infinite health. It is absolutely unfair.

There are many other issues with the tag "Balance" on GitHub, and other issues being discussed on other mediums, but none of them are anywhere near as unbalanced as this feature. If someone is unable to be killed by anything but an instant gib spell and a head on bombing, its pretty clear there is a huge balance issue present. 

tl;dr The original purpose of chemical implants has been subverted and is being abused by the worst of powergamers creating huge imbalances that make playing the game normally impossible.

Now I know I'm not supposed to post balance problems in the issue section, but it seems the only other options available to me are some sort of closed private services where I am unable to voice my opinion. If those means become available to me, then I shall post the same thing.

**Comment by XeonFaux** [comment]
Or just add a variable to the existing implants, to set them to activatable or non-activatable.

**Comment by Spacedong** [comment]
this ^

**Comment by Nebyab** [comment]
is a bug

**Comment by Pyko1** [comment]
Maybe have an advanced chem implant that can be researched at science that is in sec or med access locked box.And only that can be self activated, and remove the activation from this one?

**Comment by Spacedong** [comment]
aye^

**Comment by Sarcalogo** [comment]
I fixed it, you can close it.
